apps/storybook/src/components/ai-thinking-indicator/ai-thinking-indicator.stories.tsx

```
@@ -0,0 +1,97 @@
+import type { Meta, StoryObj } from "@storybook/react";
```

ESLint

[error] Do not import renderer package "`@storybook/react`" directly. Use a framework package instead (e.g. `@storybook/nextjs`, `@storybook/react-vite`, `@storybook/react-webpack5`, `@storybook/react-native-web-vite`, `@storybook/experimental-nextjs-vite`).

(storybook/no-renderer-packages)